mods/e2140/content/ed/infantry/a03/rules.yaml

```
@@ -0,0 +1,26 @@
+ed_infantry_a03:
+	Inherits@1: ^EdInfantry
+	Inherits@2: ^CoreArmedSelfInfantry
+	Tooltip:
+		Name: A03 Android
```

Use original name *Android A03*:

```suggestion
		Name: Android A03
```